index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Home page</title>
+        <meta charset="UTF-8">
+        <meta name="author" content="Leonardo Morganti">
+        <meta name="description" content="Gestione Competenze">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <h2>Welcome to the homepage!</h2>
```

Comment:
Scegli una lingua, sotto mi hai messo i bottoni di navigazione con contenuto in inglese

Reply:
Cambiato il testo in Italiano